util/fusermount.c

```diff
@@ -1347,6 +1347,7 @@ static void usage(void)
 	       " -V		    print version\n"
 	       " -o opt[,opt...]    mount options\n"
 	       " -u		    unmount\n"
+	       " -U 		    setup auto-unmount only\n"
```

Comment:
I also thought of re-using the combination of `--unmount` with `-o auto_unmount`.

That would have been neater, but I decided not accidentally breaking existing users of `fusermount3` was more important.